src/utils/helpers/fetchMethods.ts

```diff
@@ -0,0 +1,22 @@
+// Generic GET/POST methods for re-use throughout app
```

Comment:
Very nice to have! I wonder if it makes sense to create a folder just to store these fetch-related functions... I see that the `fetchUserInfo` could be using this helper at some point, no problem with it for now. We can change it to use when we see a commonality. I'm just thinking it would be good to also have the other fetching (with the constant to define the route) in separated files. Which is now inside the component `login`.

Reply:
> I'm just thinking it would be good to also have the other fetching (with the constant to define the route) in separated files. Which is now inside the component `login`.

Which component in `login` are you referring to?